### ISSUE TITLE
Core | Nested relatives fragments with root involved

### ISSUE THREAD

**Issue by fabienjuif** [FormidableLabs/redux-little-router#76]
Hi,

I think this is a bug.
I have those routes : 
```
export default {
  '/': {
    title: 'HOME',
    '/toto': {
      title: 'TOTO',
    },
    '/tvshow/:id': {
      title: 'TVSHOW',
    },
  },
}
```

When I use this root Component **it works** :
```
  <div>
    <Search />
    <RelativeFragment forRoute="/toto">
      <Results />
    </RelativeFragment>
  </div>
```

When i use this root Component **it doesn't work** : 
```
  <RelativeFragment forRoute="/">
    <Search />
    <RelativeFragment forRoute="/toto">
      <Results />
    </RelativeFragment>
  </RelativeFragment>
```

(I assume It works when the `Results` Component is printed when i go to `/toto`)

**Comment by tptee** [comment]
There's a pretty huge bug in `<RelativeFragment>` I found today, should have a fix in by tomorrow 👍

**Comment by fabienjuif** [comment]
Great 👍

**Comment by tptee** [comment]
Fixed in `v9.0.3`!

**Comment by fabienjuif** [comment]
Thank you for your work @tptee 
Works well here.
